content/sql/concepts/operators/terms/like/like.md

```diff
@@ -17,98 +17,31 @@ CatalogContent:
   - 'paths/design-databases-with-postgresql'
 ---
 
-Perform case-sensitive pattern matching on `TEXT` data through the use of wildcard characters.
+The `LIKE` operator returns `TRUE` if its first text argument matches the [wildcard](https://www.codecademy.com/resources/docs/sql/wildcards) pattern in its second argument.
```

Comment:
@SSwiniarski pre-maturely merged 😞 was thinking we could add the keywords "SQL `LIKE`" and/or "...SQL contains the `LIKE` operator..." to the beginning of this. Would you be able to make this change? I can also go ahead and make the change if you'd prefer 🙏 LMK

Reply:
Actually, I'll take this over this extra change and also create a `CONTAINS` entry. Please disregard!